doc/HOWTO_reservations.md

```diff
@@ -0,0 +1,103 @@
+How reservations work
+=====================
+
+Instruments have a time schedule where they can be reserved. A Reservation
+generally refer to a user scheduling the use of the instrument but also can
+refer to the instrument being offline or reserved for an administrative task.
+
+Reservations on instruments are a kind of product the user purchase and can
+have different ways of charging depending on the Instrument's pricing mode. So
+whenever a new reservation is created an order is created or updated to include
+it as part of its items.
+
+## Schedule Rules: Instruments availability schedule
```

Comment:
I think I'd rather have this with two sections: one for Daily booking and another one for the rest. I find it easier to understand as all the information related to schedule rules is in the same place